pkg/deployment/config.go

```diff
@@ -158,4 +167,7 @@ type Config struct {
 	// QueueSidecarEphemeralStorageLimit is the Ephemeral Storage Limit to set
 	// for the queue proxy sidecar container.
 	QueueSidecarEphemeralStorageLimit *resource.Quantity
+
+	// ConcurrencyStateEndpoint is the endpoint for acting when traffic drops to / increases from zero.
```

Comment:
this is a massssssive nit, but I dont like "acting" here (and somewhere above too). The contract is that this is the endpoint we will send a request to; what it chooses to do with that request (including just logging it) is up to it.

Reply:
missed a bit ^ :)